src/eddymotion/model/base.py

```diff
@@ -368,8 +385,10 @@ def fit(self, data, *args, **kwargs):
 
         self._coeff = np.array([r[0] for r in results])
 
-    def predict(self, timepoint, **kwargs):
-        """Return the *b=0* map."""
+    def predict(
+        self, timepoint, **kwargs
```

Comment:
Following the pattern proposed above:
```suggestion
        self, index, **kwargs
```

Reply:
OK. See comments in commit df51602. Resolving.